.github/workflows/api-ci.yaml

```diff
@@ -25,6 +25,10 @@ jobs:
         run: |
           npm run api::build
           npm run api::package
+      - name: Package Worker
+        run: |
+          npm run worker::build
+          npm run worker::package
       - name: CDK synth
```

Comment:
Do you think it'd be better if rename this file now that it contains the infra for both api & worker?